### DISCUSSION TITLE
> Task :react-native-reanimated:prepareHeadersForPrefab FAILED

### DISCUSSION THREAD

**Discussion by Rose-wu** [software-mansion/react-native-reanimated#6471] [Q&A]
![image](https://github.com/user-attachments/assets/9e2a0931-a1d4-404d-b52a-3c4240b2798e)
添加   "react-native-reanimated": "^3.0.0",    "@react-navigation/drawer": "^6.7.2", 
执行react-native run-android  失败  提示  Unable to delete file 'D:\wordSpace\androidProject\myProject\node_modules\react-native-reanimated\android\build\prefab-headers\reanimated' 但是 prefab-headers\文件下 没有文件 
相关版本
![image](https://github.com/user-attachments/assets/2d113f6d-0f1b-43fb-957a-2ac7aeab6e93)

**Comment by MatiPl01** [comment]
Hey! Do you see this error while trying to upgrade reanimated or after the clean install? Can you try to remove the library manually and install it again from scratch?
